docs/datamodel.rst

```diff
@@ -0,0 +1,615 @@
+                        + eventtypeid : Integer\l
+                }"
+            ]
+
+            Sagsevent -> Sag;
+            Sagsevent -> Fikspunktregisterobjekt;
+            Sagsinfo -> Sag;
+            Sagseventinfo -> Sagsevent;
+            Sagseventinfo_html -> Sagseventinfo;
+            Sagseventinfo_materiale -> Sagseventinfo;
+            Eventtype -> Sagsevent;
+        }
+
+Der findes en række sagsevents i FIRE. I tabellen herunder er de alle kort beskrevet.
+
+=========================  ============================================================================
+**Type**                   **Beskrivelse**
+-------------------------  ----------------------------------------------------------------------------
+``koordinat_beregnet``     Bruges når koordinater indsættes efter en beregning. Vil typisk resulterere
+                           i indsættelse af *n* koordinater og 1 beregning.
+``koordinat_nedlagt``      Bruges når en koordinat nedlægges.
+``observation_indsat``     Indsættelse af en eller flere observationer.
+``observation_nedlagt``    Bruges når en observation aflyses fordi den er fejlbehæftet
+``punktinfo_tilføjet``     Bruges når der tilføjes Punktinfo til et eller flere punkter
+``punktinfo_fjernet``      Bruges når Punktinfo fjernes fra et eller flere punkter
+``punkt_oprettet``         Bruges når et punkt og tilhørende geometri oprettes
+``punkt_nedlagt``          Bruges når et punkt og tilhørende geometri nedlægges. Bemærk at når et punkt
+                           nedlægges, afregistreres desuden alle tilknyttede koordinater, observationer
+                           og punktinformationer, da disse ikke længere har et tilhørsforhold.
+``kommentar``              Bruges til at tilføje fritekst kommentarer til sagen i tilfælde af at der er
+                           behov for at påhæfte sagen yderligere information som ikke passer i andre
+                           hændelser. Bruges fx også til påhæftning af materiale på sagen.
+=========================  ============================================================================
+
+Eksempel på en sag
+++++++++++++++++++
+
+Hvis man ser på en hel sag for fx en simpel kommunal vedligeholdsopgave, så vil det gå nogenlunde sådan
+her i databasen:
+
+* Opret Sag (+ Sagsinfo)
+* Opret Sagsevent af typen ``punkt_oprettet`` (+ Sagseventinfo)
+* Indsæt Punkter og Geometriobjekter, henvis til Sagsevent fra linjen ovenfor
+* Opret Sagsevent af typen ``punktinfo_tilføjet`` (+Sagseventinfo)
+* Indsæt Punktinformationer (fx ``ATTR:tabtgået``), henvis til Sagsevent fra linjen ovenfor
+* Opret Sagsevent af typen ``observation_indsat`` (+ Sagseventinfo)
+* Indsæt Observationer, henvis til Sagsevent på linjen ovenfor
+* Opret Sagsevent af typen ``koordinat_beregnet`` (+ Sagseventinfo, beregningsrapport vedlægges).
+* Indsæt Koordinater og Beregning, henvis til Sagsevent på linjen ovenfor
+* Opdater aktiv til "false" på Sagens Sagsinfo.
```

Comment:
Ikke forstået.. er aktiv et flag? Eller mener du man aktivt skal opdaterer?

Reply:
Tabellen "sagsinfo" har en kolonne der hedder "aktiv". Det felt kan have to værdier "true" og "false". Når en sag er afsluttet skal værdien for "aktiv" ændres fra "true" til "false". Det kan man være enten ved en SQL UPDATE kommando eller indsættelse af en ny række i "sagsinfo". Her gøres det altså ved at opdatere "aktiv".

Reply:
Er det bedre?

```suggestion
* Opdater aktiv-feltet til "false" på Sagens Sagsinfo.
```